src/modules/75-cd/components/PipelineStudio/DeployServiceSpecifications/PropagateWidget/PropagateWidget.tsx

```diff
@@ -70,7 +70,7 @@ export default function PropagateWidget(props: PropagateWidgetProps): JSX.Elemen
       }}
       enableReinitialize
     >
-      {formik => {
+      {(formik: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
      {(formik: unknown) => {
```